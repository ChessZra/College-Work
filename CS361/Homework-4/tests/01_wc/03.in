wc
this
is another
another test test
