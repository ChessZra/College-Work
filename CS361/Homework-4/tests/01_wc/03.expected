cs361sh> Num Bytes: 34
Num Lines: 3
Num Words: 6
cs361sh> 
